### PR TITLE
link changes

### PR DESCRIPTION
**Pull request by Randy424**
regarding: https://github.com/stolostron/backlog/issues/25128

### CODE REVIEW

**Comment by Randy424** [comment]
/hold 
Waiting for RM approval

File: frontend/src/lib/doc-util.tsx [hunk 1]
```diff
@@ -11,45 +11,44 @@ export const DOC_BASE_PATH = `${DOC_HOME}/html-single`
...
+    CREATE_CONNECTION_PROXY_AWS: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-aws`,
+    CREATE_CONNECTION_PROXY_AZURE: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-azure`,
+    CREATE_CONNECTION_PROXY_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-bare`,
+    CREATE_CONNECTION_PROXY_GCP: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-google`,
+    CREATE_CONNECTION_PROXY_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-openstack`,
+    CREATE_CONNECTION_PROXY_VIRTUALIZATION: `${DOC_BASE_PATH}/multicluster_engine/index#proxy-virtualization`,
+    CREATE_CLUSTER: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-cluster`,
+    CREATE_CLUSTER_ON_PREMISE: `${DOC_BASE_PATH}/multicluster_engine/index#creating-a-cluster-on-premises`,
+    IMPORT_CLUSTER: `${DOC_BASE_PATH}/multicluster_engine/index#importing-a-target-managed-cluster-to-the-hub-cluster`,
+    CLUSTER_SETS: `${DOC_BASE_PATH}/multicluster_engine/index#managedclustersets`,
```
Comment (KevinFCormier) [inline frontend/src/lib/doc-util.tsx:36]:
```suggestion
    CLUSTER_SETS: `${DOC_BASE_PATH}/multicluster_engine/index#managedclustersets-intro`,
```

File: frontend/src/lib/doc-util.tsx [hunk 2]
```diff
@@ -11,45 +11,44 @@ export const DOC_BASE_PATH = `${DOC_HOME}/html-single`
...
+    GLOBAL_CLUSTER_SET: `${DOC_BASE_PATH}/multicluster_engine/index#managedclustersets-intro`,
```
Comment (KevinFCormier) [inline frontend/src/lib/doc-util.tsx:37]:
```suggestion
    GLOBAL_CLUSTER_SET: `${DOC_BASE_PATH}/multicluster_engine/index#managedclustersets_global`,
```

**Pull request merged**